pkg/ovs/ovsctl/ofctl_test.go

```diff
@@ -0,0 +1,237 @@
+		for i := 0; i < len(testDumpFlows); i++ {
+			b := []byte(testDumpFlows[i])
+			for j := 0; j < len(b); j++ {
+				x = append(x, b[j])
+			}
+			x = append(x, "\n"...)
+		}
+		return x, nil
+	}
+	defer func() {
+		runCommand = runCmd
+	}()
+	out, err := client.DumpFlows()
+	assert.NoError(t, err)
+	expectedFlows := []string{"table=0, priority=200, n_packets=143, n_bytes=6006, idle_age=15512, hard_age=65534,arp actions=resubmit(,1) ", "table=1, priority=200, n_packets=12, n_bytes=504, idle_age=17282, hard_age=65534,arp,in_port=2,arp_spa=192.168.1.1,arp_sha=16:92:82:a4:69:50 actions=resubmit(,2) ", "table=2, priority=0, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534, actions=drop ", "table=3, priority=200, n_packets=29233419, n_bytes=2703471860, idle_age=8, hard_age=65534,in_port=2 actions=load:0x2->NXM_NX_REG0[0..3],resubmit(,4) ", "table=3, priority=200, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534,in_port=1 actions=load:0x1-> "}
+	assert.Equal(t, expectedFlows, out)
+}
+
+func TestDumpFlowsWithoutTableNames(t *testing.T) {
+	client := &ovsCtlClient{
+		bridge: "br-int",
+	}
+	runCommand = func(cmd string) ([]byte, error) {
+		var x = []byte{}
+		for i := 0; i < len(testDumpFlows); i++ {
+			b := []byte(testDumpFlows[i])
+			for j := 0; j < len(b); j++ {
+				x = append(x, b[j])
+			}
+			x = append(x, "\n"...)
+		}
+		return x, nil
+	}
+	defer func() {
+		runCommand = runCmd
+	}()
+	out, err := client.DumpFlowsWithoutTableNames()
+	assert.NoError(t, err)
+	expectedFlows := []string{"table=0, priority=200, n_packets=143, n_bytes=6006, idle_age=15512, hard_age=65534,arp actions=resubmit(,1) ", "table=1, priority=200, n_packets=12, n_bytes=504, idle_age=17282, hard_age=65534,arp,in_port=2,arp_spa=192.168.1.1,arp_sha=16:92:82:a4:69:50 actions=resubmit(,2) ", "table=2, priority=0, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534, actions=drop ", "table=3, priority=200, n_packets=29233419, n_bytes=2703471860, idle_age=8, hard_age=65534,in_port=2 actions=load:0x2->NXM_NX_REG0[0..3],resubmit(,4) ", "table=3, priority=200, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534,in_port=1 actions=load:0x1-> "}
+	assert.Equal(t, expectedFlows, out)
+}
+func TestDumpMatchedFlow(t *testing.T) {
+	client := &ovsCtlClient{
+		bridge: "br-int",
+	}
+	runCommand = func(cmd string) ([]byte, error) {
+		var x = []byte{}
+		for i := 0; i < len(testDumpFlows); i++ {
+			b := []byte(testDumpFlows[i])
+			for j := 0; j < len(b); j++ {
+				x = append(x, b[j])
+			}
+			x = append(x, "\n"...)
+		}
+		return x, nil
+	}
+	defer func() {
+		runCommand = runCmd
+	}()
+	matchFlow := "cookie=0xa000000000000, duration=13489003.061s, table=2, priority=0, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534, actions=drop "
+	out, err := client.DumpMatchedFlow(matchFlow)
+	assert.NoError(t, err)
+	expectedFlow := "table=2, priority=0, n_packets=0, n_bytes=0, idle_age=65534, hard_age=65534, actions=drop "
+	assert.Equal(t, expectedFlow, out)
```

Comment:
if the expectedFlow is not same as out then user must know about the test case failure etc, it makes more sense to write additional information alongside